Nesteo.Server/Controllers/Api/NestingBoxesController.cs

```diff
@@ -69,5 +71,14 @@ public async Task<ActionResult<NestingBoxPreview>> GetNestingBoxPreviewByIdAsync
 
             return nestingBoxPreview;
         }
+
+        /// <summary>
+        /// Return all inspections for a nesting box
+        /// </summary>
+        [HttpGet("{id}/inspections")]
+        public IAsyncEnumerable<Inspection> GetInspectionsByBoxIdAsync(string id)
+        {
+            return  _inspectionService.FindByBoxIdAsync(id);
+        }
     }
```

Comment:
Maybe we should add one more endpoint for retrieving the inspections as previews? @SimonOyen would probably appreciate it? :smile: 
`{id}/inspections/previews`?

Reply:
Thank you for the notes. I added the new endpoint and made the changes that you marked.